dashboard/src/api/client/axios.ts

```diff
@@ -1,21 +1,21 @@
-import axios from 'axios';
-import AxiosMockAdapter from 'axios-mock-adapter';
-import { ENV } from '../../constants/env';
-import registerLoggerInterceptor from '../interceptors/logger';
+import axios from "axios";
+import AxiosMockAdapter from "axios-mock-adapter";
+import { ENV } from "../../constants/env";
+import registerLoggerInterceptor from "../interceptors/logger";
 
 // Mock axios instance
 const mockInstance = axios.create();
 registerLoggerInterceptor(mockInstance);
 
 export const mock = new AxiosMockAdapter(mockInstance, {
-  delayResponse: 200,
+  delayResponse: 2000000,
```

Comment:
Isn't that a bit much?

Reply:
yeah, it was rolled back. Was applying some changes yesterday